cpp/frontend_pyreadline.cc

```diff
@@ -9,6 +12,8 @@
   #include <readline/readline.h>
 #endif
 
+#include "core.h"
```

Comment:
Should be `cpp/core.h` to follow the style

Also in `cpp/NINJA_subgraph.py`, it means you need a `//cpp/core` dependency (wish this would fail if you forget it -- I recently fixed other missing deps)